app/assets/javascripts/discourse/components/composer-messages.js.es6

```diff
@@ -97,17 +97,9 @@ export default Component.extend({
 
     const composer = this.composer;
     if (composer.get("privateMessage")) {
-      let usernames = composer.get("targetUsernames");
+      const recipients = composer.targetRecipientsArray;
 
-      if (usernames) {
-        usernames = usernames.split(",");
-      }
-
-      if (
-        usernames &&
-        usernames.length === 1 &&
-        usernames[0] === this.currentUser.get("username")
-      ) {
+      if (recipients.every(r => r === this.currentUser.get("username"))) {
```

Comment:
I think you also need to check for the length of recipients because `[].every(() => false) == true`